docs/reference/ml/anomaly-detection/apis/find-file-structure.asciidoc

```diff
@@ -219,6 +237,9 @@ size, which defaults to 100 Mb.
 [[ml-find-file-structure-examples]]
 ==== {api-examples-title}
 
+[[ml-find-file-structure-example-nld-json]]
+===== Ingesting newline-delimited JSON
```

Comment:
In general, we just have one example for each API.  It seems to me that if this API ceases to be "experimental" we'll want to move and expand some of these examples out of the reference page so we get explain them more thoroughly. But for now, I think it's okay like this.